boa_tester/src/main.rs

```diff
@@ -318,19 +330,43 @@ struct TestSuite {
     tests: Box<[Test]>,
 }
 
-/// Outcome of a test suite.
-#[derive(Debug, Clone, Serialize, Deserialize)]
-struct SuiteResult {
-    #[serde(rename = "n")]
-    name: Box<str>,
-    #[serde(rename = "c")]
+/// Represents a tests statistic
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+struct Statistics {
+    #[serde(rename = "t")]
     total: usize,
     #[serde(rename = "o")]
     passed: usize,
     #[serde(rename = "i")]
     ignored: usize,
     #[serde(rename = "p")]
     panic: usize,
+}
+
+impl Add for Statistics {
+    type Output = Self;
+
+    fn add(self, rhs: Self) -> Self::Output {
+        Self {
+            total: self.total + rhs.total,
+            passed: self.passed + rhs.passed,
+            ignored: self.ignored + rhs.ignored,
+            panic: self.panic + rhs.panic,
+        }
+    }
+}
+
+/// Outcome of a test suite.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+struct SuiteResult {
+    #[serde(rename = "n")]
+    name: Box<str>,
+    #[serde(rename = "a", default)]
```

Comment:
This doesn't need a `default`, because all schemas need to have an `a` for them to be valid. We're making an exception for `a5` and `a6` because they were introduced later.